jdi-light/src/main/java/com/epam/jdi/light/elements/init/InitActions.java

```diff
@@ -63,7 +63,7 @@ static void webPageSetup(SiteInfo info) {
         $(IsDropdown.class, Dropdown.class)
     );
     public static MapArray<String, InitRule> INIT_RULES = map(
-        $("WebList", iRule(f -> isList(f, WebElement.class), info -> new WebList())),
+        $("WebList", iRule(f -> isList(f, WebElement.class), info -> new WebList(0))),
```

Comment:
could we write something like that: new WebList() should understand that it works with WebElement so there is no need to pass 0 inside  ? (@RomanIovlev )

Reply:
For me it's the place itself where WebList understands this and calls selenium realisation afterwards.